flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/utils/SourceCodeGenerator.java

```diff
@@ -422,4 +395,47 @@ private void generateInferredGraphqlClassField(Function function, StringBuilder
         stringBuilder.append(" ").append(function.name());
         stringBuilder.append(";");
     }
+
+    private String generateGraphqlClassResourceFunction(Function function) {
```

Comment:
can make these methods static

Reply:
the name of this function should not have graphql. Because we not doing anything specific to graphql.

Reply:
> can make these methods static
My idea: non-static methods provides more flexibility for future changes if we want to access instance state.